site/content/operational_guide/resource_limits.md

```diff
@@ -94,6 +94,34 @@ limits:
   maxOutstandingReadRequests: 0
 ```
 
+### Dynamic configuration
+
+Query limits can be dynamically driven by etcd to adjust limits without redeploying. By updating the `m3db.query.limits` key in etcd, specific limits can be overriden. M3Coordinator exposes an API for updating etcd key/value pairs and so this API can be used for modifying these dynamic overrides. For example,
+
+```
+curl -vvvsSf -X POST 0.0.0.0:7201/api/v1/kvstore -d '{
+  "m3db.query.limits",
+  "value":{
+    "maxRecentlyQueriedSeriesDiskBytesRead": {
```

Comment:
nit; list the available query types here since it's a bit annoying having to dig through proto to find these field names